packages/dashboard/src/router/AppRoutes.tsx

```
@@ -1,66 +1,88 @@
+const SQLEditorLayout = lazy(() => import('#features/database/components/SQLEditorLayout'));
+const BackupsPage = lazy(() => import('#features/database/pages/BackupsPage'));
+const DatabaseFunctionsPage = lazy(() => import('#features/database/pages/FunctionsPage'));
+const IndexesPage = lazy(() => import('#features/database/pages/IndexesPage'));
+const MigrationsPage = lazy(() => import('#features/database/pages/MigrationsPage'));
+const PoliciesPage = lazy(() => import('#features/database/pages/PoliciesPage'));
+const SQLEditorPage = lazy(() => import('#features/database/pages/SQLEditorPage'));
+const TablesPage = lazy(() => import('#features/database/pages/TablesPage'));
+const TemplatesPage = lazy(() => import('#features/database/pages/TemplatesPage'));
+const TriggersPage = lazy(() => import('#features/database/pages/TriggersPage'));
+const DeploymentsLayout = lazy(() => import('#features/deployments/components/DeploymentsLayout'));
+const DeploymentDomainsPage = lazy(() => import('#features/deployments/pages/DeploymentDomainsPage'));
```

ESLint

[error] Replace `()·=>·import('`#features/deployments/pages/DeploymentDomainsPage`')` with `⏎··()·=>·import('`#features/deployments/pages/DeploymentDomainsPage`')⏎`

(prettier/prettier)